File: filedata/entity/c#/Videos.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("videos")]
    public class Videos
    {
        
        /// <summary>
        /// 视频ID
        /// </summary>
        [Column("Id")]
        public int Id { get; set; }
    
    
        /// <summary>
        /// 视频名称
        /// </summary>
        [Column("VideoName")]
        public string VideoName { get; set; }
    
    
        /// <summary>
        /// 视频类型 1=长视频 2=专辑视频
        /// </summary>
        [Column("VideoType")]
        public int VideoType { get; set; }
    
    
        /// <summary>
        /// 视频封面
        /// </summary>
        [Column("VideoCover")]
        public string VideoCover { get; set; }
    
    
        /// <summary>
        /// 视频播放链接
        /// </summary>
        [Column("VideoPlayConnection")]
        public string VideoPlayConnection { get; set; }
    
    
        /// <summary>
        /// 视频预览链接
        /// </summary>
        [Column("VideoPreviewConnection")]
        public string VideoPreviewConnection { get; set; }
    
    
        /// <summary>
        /// 视频下载链接
        /// </summary>
        [Column("VideoDownloadLink")]
        public string VideoDownloadLink { get; set; }
    
    
        /// <summary>
        /// 视频点赞次数
        /// </summary>
        [Column("VideoLikes")]
        public int VideoLikes { get; set; }
    
    
        /// <summary>
        /// 视频观看次数
        /// </summary>
        [Column("VideoWatchCount")]
        public int VideoWatchCount { get; set; }
    
    
        /// <summary>
        /// 视频内容标签 
 如: 偷拍,SM,后入,人妻 等 
使用#号分割
        /// </summary>
        [Column("VideoTag")]
        public string VideoTag { get; set; }
    
    
        /// <summary>
        /// 视频专属公司 不为null则是专属视频
        /// </summary>
        [Column("VideoCompanyStyle")]
        public string VideoCompanyStyle { get; set; }
    
    
        /// <summary>
        /// 是否收费视频 0=免费 1=收费
        /// </summary>
        [Column("IsCharge")]
        public bool IsCharge { get; set; }
    
    
        /// <summary>
        /// 收费价格
        /// </summary>
        [Column("ChargeAmount")]
        public decimal ChargeAmount { get; set; }
    
    
        /// <summary>
        /// 视频总时间  /  分钟
        /// </summary>
        [Column("VideoTime")]
        public int VideoTime { get; set; }
    
    
        /// <summary>
        /// 视频创建时间
        /// </summary>
        [Column("CreatTime")]
        public DateTime CreatTime { get; set; }
    
    
        /// <summary>
        /// 视频最后修改时间
        /// </summary>
        [Column("UpdateTime")]
        public DateTime UpdateTime { get; set; }
    
    
        /// <summary>
        /// 最后修改人
        /// </summary>
        [Column("Modifier")]
        public string Modifier { get; set; }
    
    
        /// <summary>
        /// 视频从属ID
VideoType=1则从属于VideosType(视频类型)表  VideoType=2则从属于AlbumType(专辑类型)表
        /// </summary>
        [Column("VideoTypeId")]
        public string VideoTypeId { get; set; }
    
    
        /// <summary>
        /// 视频详情描述
        /// </summary>
        [Column("VideoTitle")]
        public string VideoTitle { get; set; }
    
    
    }
}

File: filedata/entity/java/videos.java

package entity;
// 
public class Videos  extends BaseDomain{
    public Videos() {

    }
    
    //
    private bigint(20) unsigned Id;
        
    //
    private String VideoName;
        
    //
    private int(255) unsigned VideoType;
        
    //
    private String VideoCover;
        
    //
    private String VideoPlayConnection;
        
    //
    private String VideoPreviewConnection;
        
    //
    private String VideoDownloadLink;
        
    //
    private int(255) VideoLikes;
        
    //
    private int(255) VideoWatchCount;
        
    //
    private String VideoTag;
        
    //
    private String VideoCompanyStyle;
        
    //
    private bit(1) IsCharge;
        
    //
    private double ChargeAmount;
        
    //
    private int(255) unsigned VideoTime;
        
    //
    private Date CreatTime;
        
    //
    private Date UpdateTime;
        
    //
    private String Modifier;
        
    //
    private String VideoTypeId;
        
    //
    private String VideoTitle;
        
    public bigint(20) unsigned getId() {
        return Id;
    }
    public void setId(bigint(20) unsigned Id) {
        this.Id = Id;
    }
            
    public String getVideoname() {
        return Videoname;
    }
    public void setVideoname(String Videoname) {
        this.Videoname = Videoname;
    }
            
    public int(255) unsigned getVideotype() {
        return Videotype;
    }
    public void setVideotype(int(255) unsigned Videotype) {
        this.Videotype = Videotype;
    }
            
    public String getVideocover() {
        return Videocover;
    }
    public void setVideocover(String Videocover) {
        this.Videocover = Videocover;
    }
            
    public String getVideoplayconnection() {
        return Videoplayconnection;
    }
    public void setVideoplayconnection(String Videoplayconnection) {
        this.Videoplayconnection = Videoplayconnection;
    }
            
    public String getVideopreviewconnection() {
        return Videopreviewconnection;
    }
    public void setVideopreviewconnection(String Videopreviewconnection) {
        this.Videopreviewconnection = Videopreviewconnection;
    }
            
    public String getVideodownloadlink() {
        return Videodownloadlink;
    }
    public void setVideodownloadlink(String Videodownloadlink) {
        this.Videodownloadlink = Videodownloadlink;
    }
            
    public int(255) getVideolikes() {
        return Videolikes;
    }
    public void setVideolikes(int(255) Videolikes) {
        this.Videolikes = Videolikes;
    }
            
    public int(255) getVideowatchcount() {
        return Videowatchcount;
    }
    public void setVideowatchcount(int(255) Videowatchcount) {
        this.Videowatchcount = Videowatchcount;
    }
            
    public String getVideotag() {
        return Videotag;
    }
    public void setVideotag(String Videotag) {
        this.Videotag = Videotag;
    }
            
    public String getVideocompanystyle() {
        return Videocompanystyle;
    }
    public void setVideocompanystyle(String Videocompanystyle) {
        this.Videocompanystyle = Videocompanystyle;
    }
            
    public bit(1) getIscharge() {
        return Ischarge;
    }
    public void setIscharge(bit(1) Ischarge) {
        this.Ischarge = Ischarge;
    }
            
    public double getChargeamount() {
        return Chargeamount;
    }
    public void setChargeamount(double Chargeamount) {
        this.Chargeamount = Chargeamount;
    }
            
    public int(255) unsigned getVideotime() {
        return Videotime;
    }
    public void setVideotime(int(255) unsigned Videotime) {
        this.Videotime = Videotime;
    }
            
    public Date getCreattime() {
        return Creattime;
    }
    public void setCreattime(Date Creattime) {
        this.Creattime = Creattime;
    }
            
    public Date getUpdatetime() {
        return Updatetime;
    }
    public void setUpdatetime(Date Updatetime) {
        this.Updatetime = Updatetime;
    }
            
    public String getModifier() {
        return Modifier;
    }
    public void setModifier(String Modifier) {
        this.Modifier = Modifier;
    }
            
    public String getVideotypeid() {
        return Videotypeid;
    }
    public void setVideotypeid(String Videotypeid) {
        this.Videotypeid = Videotypeid;
    }
            
    public String getVideotitle() {
        return Videotitle;
    }
    public void setVideotitle(String Videotitle) {
        this.Videotitle = Videotitle;
    }
             
    @Override
    public String toString() {
        return "videos[Id=" + Id + ",VideoName=" + VideoName + ",VideoType=" + VideoType + ",VideoCover=" + VideoCover + ",VideoPlayConnection=" + VideoPlayConnection + ",VideoPreviewConnection=" + VideoPreviewConnection + ",VideoDownloadLink=" + VideoDownloadLink + ",VideoLikes=" + VideoLikes + ",VideoWatchCount=" + VideoWatchCount + ",VideoTag=" + VideoTag + ",VideoCompanyStyle=" + VideoCompanyStyle + ",IsCharge=" + IsCharge + ",ChargeAmount=" + ChargeAmount + ",VideoTime=" + VideoTime + ",CreatTime=" + CreatTime + ",UpdateTime=" + UpdateTime + ",Modifier=" + Modifier + ",VideoTypeId=" + VideoTypeId + ",VideoTitle=" + VideoTitle + "]";
    }
        
    
}
        